docs/writing-docs/tutorials/activities.md

```diff
@@ -66,7 +66,7 @@ In an activity style tutorial, the title can appear in a psuedo-activity that se
 
 ## Introduction
 
-### Introduction step @unplugged
+### Introduction step @showhint
```

Comment:
I think unplugged -> showDialog, and fullscreen -> showHint?? but I could be wrong XD

Reply:
No, you are correct. Thanks.